scripts/eurostat/life_expectancy_by_age_gender/README.md

```diff
@@ -0,0 +1,78 @@
+
+# Importing Eurostat Life Expectancy By Age, Sex (demo_r_mlifexp)  Into Data Commons
+
+Author: qlj-lijuan
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Artifacts](#import-artifacts)
+1. [Generating Artifacts](#generating-artifacts)
+
+## About the Dataset
+
+### Download URL
+
+tsv file is available for download from https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/demo_r_mlifexp.tsv.gz.
+
+### Overview
+
+This dataset reports life expectancy by age, gender in countries and NUTS2 regions in Europe.
+The dataset is broken into 2 parts: the country level and NUT2 regions for the convenience of importing into DataCommons
+
+This dataset contains the following info:
```

Comment:
I think your 2 sentence summary above is enough, no need for this bullet list.

Reply:
done.